.gitignore

```diff
@@ -4,5 +4,4 @@ vendor
 /.editorconfig
 /.gitattributes
 default-site/files
-modules/contrib
```

Comment:
The `contrib` directory is now explicitly excluded from our host-to-container mapping, so we don't need to exclude it from git anymore.